src/main/java/io/kestra/plugin/linkedin/CommentTrigger.java

```diff
@@ -0,0 +1,322 @@
+        List<String> renderedPostUrns = runContext.render(this.postUrns).asList(String.class);
+        String renderedLinkedinVersion = runContext.render(this.linkedinVersion).as(String.class).orElse("202502");
+
+        HttpRequestFactory requestFactory = createLinkedinHttpRequestFactory(renderedAccessToken);
+
+        List<String> postsToMonitor = new ArrayList<>();
+        if (this.postUrns != null) {
+            postsToMonitor.addAll(renderedPostUrns);
+        }
+
+        if (postsToMonitor.isEmpty()) {
+            runContext.logger().warn("No post URNs provided to monitor");
+            return Optional.empty();
+        }
+
+        runContext.logger().info("Checking for new comments on {} posts", postsToMonitor.size());
+
+        Instant lastCheckTime = context.getNextExecutionDate() != null
+            ? context.getNextExecutionDate().toInstant().minus(this.interval)
+            : Instant.now().minus(this.interval);
+
+        List<CommentData> newComments = new ArrayList<>();
+        Gson gson = new Gson();
+
+        try {
+            for (String postUrn : postsToMonitor) {
+                String encodedUrn = URLEncoder.encode(postUrn, StandardCharsets.UTF_8);
+                String apiUrl = "https://api.linkedin.com/rest/socialActions/" + encodedUrn + "/comments";
+
+                GenericUrl url = new GenericUrl(apiUrl,true);
+                HttpRequest request = requestFactory.buildGetRequest(url);
+
+                request.getHeaders().set("LinkedIn-Version", renderedLinkedinVersion);
+                request.getHeaders().set("X-Restli-Protocol-Version", "2.0.0");
+
+                HttpResponse response = request.execute();
+                String responseBody = response.parseAsString();
+
+                JsonObject jsonResponse = gson.fromJson(responseBody, JsonObject.class);
+
+                if (jsonResponse.has("elements")) {
+                    JsonArray elements = jsonResponse.getAsJsonArray("elements");
+
+                    for (JsonElement element : elements) {
+                        JsonObject commentObj = element.getAsJsonObject();
+                        CommentData comment = parseCommentData(postUrn, commentObj, lastCheckTime);
+
+                        if (comment != null) {
+                            newComments.add(comment);
+                        }
+                    }
+                }
+            }
+
+            if (newComments.isEmpty()) {
+                runContext.logger().info("No new comments found since last check");
+                return Optional.empty();
+            }
+
+            runContext.logger().info("Found {} new comments", newComments.size());
+
+            // Get the most recent comment for the output
+            CommentData latest = newComments.stream()
+                .max((c1, c2) -> c1.getCreatedTime().compareTo(c2.getCreatedTime()))
+                .orElse(newComments.get(0));
```

Comment:
Use `getFirst` instead of `get(0)` everywhere

```suggestion
                .orElse(newComments.getFirst());
```